test/junit/src/test/scala/strawman/collection/mutable/ArraySeqTest.scala

```diff
@@ -0,0 +1,44 @@
+package strawman.collection.mutable
```

Comment:
Why is it in the `mutable` package?

Reply:
I copied the existing test for `Vector` that already exist in the `mutable` package.

Reply:
Oh that’s a mistake. I’ll fix that one :)